src/pages/messages/TicketAccordion.tsx

```diff
@@ -39,7 +39,11 @@ export const TicketAccordion = ({ ticket }: TicketAccordionProps) => {
       const newMessage: Message = {
         ...messagesCopy[0],
         createdDate: new Date().toString(),
-        sender: username ?? '',
+        sender: {
+          firstName: user?.givenName ?? '?',
```

Comment:
Skal det tas hensyn til `preferred names` her?

Reply:
Burde sjekke for preferred ja, will fix

> Ja, vi har objekt som kan ha de fire navn feltene, og dersom objektet har de fire feltene (fra cristin respons) , får de verdi, dersom objektet har kun firstName og lastName, vil prefferedNames få null verdi, og blir ikke med i json som serveres til dere (frontend)